From b2dd9428ee44290b4cd75c1612b6f0794541c78b Mon Sep 17 00:00:00 2001
From: keita kobayashi <keita.kobayashi.ym@renesas.com>
Date: Thu, 29 May 2014 16:24:52 +0900
Subject: ARM: shmobile: r8a7791: Support Core-Standby for Suspend to RAM

Add r8a7791 Core-Standby state for Suspend to RAM support.

Signed-off-by: Keita Kobayashi <keita.kobayashi.ym@renesas.com>
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
(cherry picked from commit 7f6234013a835476f1503be2c9287f1fe3497457)
Signed-off-by: Simon Horman <horms+renesas@verge.net.au>
---
 arch/arm/mach-shmobile/smp-r8a7791.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm/mach-shmobile/smp-r8a7791.c b/arch/arm/mach-shmobile/smp-r8a7791.c
index 17720860f0dd..c6543b6ec759 100644
--- a/arch/arm/mach-shmobile/smp-r8a7791.c
+++ b/arch/arm/mach-shmobile/smp-r8a7791.c
@@ -52,6 +52,7 @@ static void __init r8a7791_smp_prepare_cpus(unsigned int max_cpus)
 	iounmap(p);
 
 	r8a7791_pm_init();
+	shmobile_smp_apmu_suspend_init();
 }
 
 static int r8a7791_smp_boot_secondary(unsigned int cpu,
-- 
2.1.2

